src/common/utxobased/db/makeProcessor.ts

```diff
@@ -247,9 +247,9 @@ export async function makeProcessor(
         const processorTx:
           | IProcessorTransaction
           | undefined = await tables.txById
-            .query('', [tx.txid])
-            .then(transactions => transactions[0])
-            .catch(_ => undefined)
+          .query('', [tx.txid])
+          .then(transactions => transactions[0])
+          .catch(_ => undefined)
```

Comment:
Fixup here: Don't add the spaces in the previous commit only to remove them here.